tpl/crawler/settings-general.tpl.php

```diff
@@ -28,6 +28,25 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_CRAWLER_SCHEDULE_TIME; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_input($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('Change the crawler running time.', 'litespeed-cache'); ?>
+					<br />
+					<?php echo sprintf( __('You can add multiple times delimited by %s', 'litespeed-cache'), '<code>,</code>'); ?>
```

Comment:
`You can add multiple ranges in 24-hour format ("HH:mm-HH:mm") delimited by`
(If that time format is incorrect, you can change it. The point is that we should be explicit about what a start-end range looks like)

Reply:
@lslisa changed text